boards/arm/verdin_imx8mm_m4/verdin_imx8mm_m4.yaml

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2023, Toradex
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+identifier: verdin_imx8mm_m4
+name: Toradex Verdin i.MX8M Mini M4
+type: mcu
+arch: arm
+ram: 32
+flash: 32
+toolchain:
+  - zephyr
+  - gnuarmemb
+  - xtools
+testing:
```

Comment:
> is ignore_tags needed to resolve a CI failure? If not, I'd like to remove it

I think so, I'd keep to prevent any errors in this regard.

> Can we add a supported block here? based on board documentation, I believe we need the following:

Nice, added, thanks!